src/smartdc/lib/ntp_config.js

```diff
@@ -1,6 +1,11 @@
 #!/usr/node/bin/node
 /* vim: syn=javascript ts=8 sts=8 sw=8 noet: */
 
+/*
```

Comment:
Also, is this under some sort of license (CDDL1.0 or MPL2.0)?

Reply:
Added CDDL license header.

Reply:
Also, CDDL vs MPL here:

The reason for CDDL, with smartos-live, the convention has been for files that natively live in the repo (i.e., are not copied in from elsewhere), things tend to be CDDL. This file predates Joyent's significant use of MPL, so it seems the original intention would have been CDDL.